docs/components/filter.md

```diff
@@ -3,24 +3,134 @@ layout: layouts/component.njk
+
+User research will tell you the best order of filters.
+### Navigating filters
+#### Show and hide individual filters
+
+If you're providing multiple ways of filtering, you may want to let users show and hide filters as needed.
+
+<div class="govuk-form-group">
+<table>
+    <tr>
+        <td>
+            <img src="../../assets/images/filters-01.png" alt="" width="100%">
+        </td>
+        <td>
+            <img src="../../assets/images/filters-02.png" alt="" width="100%">
+        </td>
+        <td>
+            <img src="../../assets/images/filters-03.png" alt="" width="100%">
+        </td>
+    </tr>
+</table>
+</div>
+
+User research will tell you if users need filters they have opened to stay open when they return to the page.
+
+If you're letting users show and hide individual filters you must:
+
+- use clear labels to help users understand what options are in the filter
+- allow the user to navigate with a keyboard
+- inform the user where they are and when content changes
+
+If you struggle to come up with clear labels, it might be because the way you’ve separated the content is not clear.
+
+#### Reducing the list of filter options
+
+If a filter has a long list of options, use an [accessible autocomplete](https://github.com/alphagov/accessible-autocomplete) (type ahead) to let users narrow down the options before making a selection.
+
+You should only show options if they match what the user has typed, of if they are already selected.
 
-The filter component can consist of any form control like radio buttons, checkboxes, date inputs and text boxes.
+You should make it easier for users to narrow the options by:
 
-Users can select 1 or more filters and submit the form. The page refreshes to show the items that match the filters. The selected filters are also marked at the top of the filter to let users see what they've selected and remove them easily.
+- making it case insensitive
+- stripping out punctuation characters and duplicate whitespace
+- seeing ‘&’ and ‘and’ as the same
 
-Clicking on a selected filter, refreshes the page and removes the filter.
+<div class="govuk-form-group">
+<table>
+    <tr>
+        <td>
+            <img src="../../assets/images/filters-04.png" alt="" width="100%">
+        </td>
+        <td>
+            <img src="../../assets/images/filters-05.png" alt="" width="100%">
+        </td>
+    </tr>
+</table>
+</div>
+
+## Research
+
+Users don't always see they can filter. Some users try to filter from the columns headings Sirius for OPG and Manage Offenders in Custody for HMPPS)
+
+This component:
+
+- has been usability tested
+- is working in a live service (Sirius for Office of the Public Guardian)
+
+Using the native date picker as a filter:
+
+- tested well with users
+- is accessible in all browsers except Firefox
```

Comment:
Sorry, I wasn't clear with my previous feedback: we didn't find the date picker to be inaccessible with users, they just had usability concerns: you couldn't copy-paste—which you could with our previous JavaScript solution—and entering historic dates was hard, though that UI may have changed now.

I was more meaning to challenge whether it's fair to say it "tested well with users" as a blanket statement, since we also have experience suggesting it didn't in some circumstances.

Reply:
Have removed this section.